src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -136,9 +140,9 @@ for (const theme of ['refresh', 'classic']) {
             onSplitPanelPreferencesChange={noop}
           />
         );
-        act(() => wrapper.findSplitPanel()!.findOpenButton()!.click());
-        act(() => wrapper.findSplitPanel()!.findCloseButton()!.click());
-        expect(wrapper.findSplitPanel()!.findOpenButton()!.getElement()).toHaveFocus();
+        wrapper.findSplitPanelOpenButton()!.click();
```

Comment:
When split panel is closed, `findSplitPanel` may return null. But the open button is available because it is rendered outside, directly in app layout. To open that panel, need to skip `findSplitPanel` and use `findSplitPanelOpenButton`

We have got a few tests relying on this behavior, so I updated them to use the new test-util